src/components/Dashboard/MySkills.js

```diff
@@ -153,6 +167,33 @@ class MySkills extends Component {
                 {skillsData.map((skill, index) => {
                   return (
                     <TableRow key={index}>
+                      <TableRowColumn>
+                        <Link
+                          to={{
+                            pathname:
+                              '/' +
+                              skill.group +
+                              '/' +
+                              skill.skill_name
```

Comment:
Use `skill_tag` here. Also, in the link of Skill Name.

Reply:
Fixed